### PR TITLE
Update syllabus for new semester

### PR DESCRIPTION
**Pull request by ethanwhite**
Includes some additional minor cleanup and improvement

### CODE REVIEW

File: syllabus.md [hunk 1]
```diff
@@ -183,23 +181,22 @@ of the assignment as you can based on what we have covered in class by the
...
 
 ### Course Technology
 
-Students are required to provide their laptops and to install free and open
+Students are required to provide their own laptops and to install free and open
 source software on those laptops. Support will be provided by the instructor in
-the installation of required software.
+the installation of required software. If you don't have access to a laptop
+please contact the instructor and they will do their best to provide you with
+one.
```
Comment (brymz) [inline syllabus.md]: You could provide a link to the setup page, here.

**Comment by brymz** [comment]
I left one minor comment. Otherwise, it's ready to merge.

File: syllabus.md (hunk 1 above)
Comment (ethanwhite) [inline syllabus.md]: Done.

Comment (brymz) [inline syllabus.md]: Sorry, I think it's `{{ site.baseurl }}/computer-setup`. Also, we will want to make it clear somewhere (here or the setup page) that semester-class students need to setup SQL and R.

Comment (ethanwhite) [inline syllabus.md]: Thanks. Apparently my Jekyll's gotten a little rusty :smile:. Let's add that note on the setup page and I'll let you take care of that if you have time.

Comment (ethanwhite) [inline syllabus.md]: Should be fixed now.

Comment (brymz) [inline syllabus.md]: A small victory for the GitHub work flow.

**Pull request merged**